gctscript/modules/ta/indicators/indicators.go

```diff
@@ -0,0 +1,61 @@
+package indicators
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/thrasher-corp/gct-ta/pkg/indicators"
+	"github.com/thrasher-corp/gocryptotrader/gctscript/modules"
+)
+
+// OHLCV locale string for OHLCV data conversion failure
```

Comment:
locale -> local

Reply:
this is locale as in language localisation

Reply:
Okay! I have never associated the term before. Will resolve.